### ISSUE TITLE
Trajectory issue: Min value out of [-0.5,0.5] range

### ISSUE THREAD

**Issue by dennishnf** [mind-inria/mri-nufft#72]
After creating 3D cones trajectories, I have observed that he min value of the trajectory if out f the [-0.5,0.5] range. Here I attached the code to reproduce this issue:

```
import mrinufft

# Trajectory parameters
Nc = int(208*384/10)  # Number of shots
Ns = 384 # Number of samples per shot
in_out = False  # Choose between in-out or center-out trajectories
tilt = "uniform"  # Angular distance between shots

trajectory = mrinufft.initialize_3D_cones(Nc, Ns, in_out=in_out)

trajectory = trajectory.reshape(Nc*Ns,3)

# here min value out of [-0.5,0.5] range
trajectory.min()
-0.5000150076299451
```

**Comment by chaithyagr** [comment]
I think some of the trajectories will slightly exceed the constraints just to ensure we still satisfy the hardware constraints. I will let @Daval-G comment if this is a real bug to be fixed. Else, we can close this.

**Comment by paquiteau** [comment]
Addditionally we might want to partially change the behaviour of [`proper_trajectory`](https://github.com/mind-inria/mri-nufft/blob/eda0e6298adf2b7f2c6efbd3e9fe208d978528f3/src/mrinufft/_utils.py#L9):
- currently if the trajectory lies in a range extended by epsilon [^1] the data is not rescale and used as it. 
- We could either truncate (or wrap ?) coordinates that are outside the range, but inside the epsilon region 

[^1]: epsilon = 1e-4, it's hardcoded for now.

**Comment by Daval-G** [comment]
This problem comes from a quite approximative initialization of the cones specific to 3D Cones, and particularly the way their width were computed. It should be fixed in PR #86 because the 3D Cones trajectory now relies on our own tools and 2D spirals, along with a calculation of the cone widths based on optimal surface sphere packing.

**Comment by Daval-G** [comment]
Based on the example provided, it seems that the problem is fixed. Can you confirm it's OK on your side @dennishnf ?

Quick note: I don't recommend using uniform tilt for 3D Cones, the default one is golden tilt because it allows the cones to match a Fibonacci lattice of the 3D k-space sphere, which is approximately well-balanced.
